test/jdk/java/awt/Multiscreen/MultiScreenInsetsTest/MultiScreenInsetsTest.java

```diff
@@ -79,14 +84,42 @@ public static void main(String[] args) throws InterruptedException {
 
             Rectangle frameBounds = frame.getBounds();
             frame.dispose();
+
+            /*
+             * On Windows, the top-left corner of an undecorated maximized frame may have negative coordinates (x, y).
+             * Adjust the frame bounds accordingly.
+             */
+            if (frameBounds.x < bounds.x)
+            {
```

Comment:
```suggestion
            if (frameBounds.x < bounds.x) {
```
Use Java code style.